typescript/stalin-sort.ts

```diff
@@ -0,0 +1,12 @@
+const stalinSort = (arr: number[]) => {
```

Comment:
Please, transform this arrow function in a common function and put the returning type as number[]